docs/UserGuide.md

```diff
@@ -248,6 +246,12 @@ Example:
 You can key `list` to see the sequence of tasks in the list to check the index of the task you wish to edit.
 </div>
 
+<div markdown="span" class="alert alert-warning">:exclamation: **Warning:**
+You should include the existing tag(s) in the command when adding a new tag. Otherwise, existing tags will be overwritten.<br>
+e.g. `edit 1 t/CS2103T t/Tutorial`, where `CS2103T` is an existing tag and `Tutorial` is a new tag to be added.
```

Comment:
should it be explained that index 1 means that the first task listed in the task list will be edited?